### ISSUE TITLE
Support selection of multiple interval?

### ISSUE THREAD

**Issue by kanitw** [vega/vega-lite#6548]
It may make sense to have the ability to shift click and select multiple intervals.

**Comment by dwootton** [comment]
Multiple interval selections would be super useful. Just ran into a use case where I was attempting to select the tail-ends of distributions. Having the ability to shift-drag to add a new interval selection would have been exactly what I needed.

**Comment by domoritz** [comment]
Do you need an arbitrary number of interval selections or just n>1?

**Comment by dwootton** [comment]
Arbitrary numbers would likely be ideal as I can imagine plenty of use cases, but with my case, n=2 would suffice.

**Comment by domoritz** [comment]
Agreed. If you have n=2, you can make two separate selections with different modifier keys.
